runtime/restart/monitor/change.go

```diff
@@ -63,11 +55,58 @@ func (s *startChange) apply(ctx context.Context, client *containerd.Client) erro
 		}
 	}
 	killTask(ctx, s.container)
-	task, err := s.container.NewTask(ctx, log)
+
+	spec, err := s.container.Spec(ctx)
 	if err != nil {
 		return err
 	}
-	return task.Start(ctx)
+
+	useTTY := spec.Process.Terminal
+
+	var con console.Console
+	if useTTY {
+		con = console.Current()
+		defer con.Reset()
+		if err := con.SetRaw(); err != nil {
+			return err
+		}
+	}
+
+	task, err := tasks.NewTask(ctx, client, s.container, "", con, false, s.logURI, []cio.Opt{})
```

Comment:
We might just need to supply the terminal bool and leave the rest of the io logic the same, I'm not certain though (e.g. use loguri if its supplied, otherwise null). Something like this, where we can use NewCreator to build up what we need https://github.com/containerd/containerd/blob/main/integration/client/container_test.go#L2671